sufia-models/lib/sufia/models/engine.rb

```diff
@@ -31,6 +31,7 @@ class Engine < ::Rails::Engine
       config.activity_to_show_default_seconds_since_now = 24 * 60 * 60
       config.arkivo_api = false
       config.retry_unless_sleep = 0.0
+      config.geonames_username = "cam156"
```

Comment:
I think we should default this value to an empty string. This will be a breaking change, but I don't think all sufia instances should be using the same config value. Thanks for finding this and working on it, @tonmoy18